bsoid_app.py

```diff
@@ -9,70 +9,46 @@
 
 def streamlit_run(pyfile):
     os.system("streamlit run {}.py".format(pyfile))
-
+working_dir = os.environ.get('working_dir', '')
```

Comment:
I'd set these (and other environment variables) up at the top of the file and capitalize the ones that be constants. So maybe `working_dir` is not? But `software_choice` is so it would be `SOFTWARE_CHOICE`

Reply:
moved to top and renamed as WORKING_DIR_BSOID